projects/components/src/form/form-checkbox/form-checkbox.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="form-group">
-    <div class="clr-form-control clr-row" [ngClass]="{ 'clr-form-control-disabled': disabled }">
-        <label class="clr-control-label clr-col-12 clr-col-md-2" [for]="id" *ngIf="label">{{ label }}</label>
-        <div class="clr-control-container clr-col-12 clr-col-md-10" [ngClass]="{ 'clr-error': showErrors }">
+    <div class="clr-form-control" [ngClass]="{ 'clr-form-control-disabled': disabled }">
```

Comment:
vcdResponsiveInput is missing?

Reply:
How did I miss this? Does that mean I didn't properly test it? 🤦